CHANGELOG.md

```diff
@@ -1,5 +1,41 @@
 # Change Log
 
+## 7.24.0 (June 19, 2018)
+
+- Enhancement: Associated `plist` with `config` icon. ([@JimiC](https://github.com/JimiC) in [#1630](https://github.com/vscode-icons/vscode-icons/pull/1630))
+- Misc: Disable `before_script` step in docker stages. ([@JimiC](https://github.com/JimiC) in [#1629](https://github.com/vscode-icons/vscode-icons/pull/1629))
```

Comment:
Until now we've used `Feature`, `Fix` and `Enhancement` for everything. I think that we can put all those `Misc` logs into one of those three categories.